theme/admin.py

```diff
@@ -16,9 +18,32 @@ class HomePageAdmin(PageAdmin):
     inlines = (IconBoxInline,)
 
 
+class UserQuotaForm(forms.ModelForm):
+    user = forms.ModelChoiceField(queryset=User.objects.exclude(is_superuser=True))
+
+    class Meta:
+        model = UserQuota
+        fields = ['user', 'allocated_value', 'used_value', 'unit', 'zone']
+
+    def save(self, *args, **kwargs):
+        instance = super(UserQuotaForm, self).save(commit=False)
+        instance.user = self.cleaned_data['user']
+        return instance
```

Comment:
@pkdash I initially had save() call after line#30, however, it raised exceptions, after some debugging, I realized it should not be saved here in the form save method since it is actually saved in save_model() method in the Django admin.ModelAdmin class. After removing save() call here, everything works and it is saved correctly later by save_model() method. This form save() method just returns the UserQuota model instance with user foreign key field set properly.